src/components/editor/PropertyResourceTemplate.jsx

```diff
@@ -12,50 +12,65 @@ class PropertyResourceTemplate extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      collapse: false
+      collapse: false,
+      output: this.populatePropertyTemplates()
     }
   }
 
   handleAddClick = (event) => {
-     event.preventDefault()
+    event.preventDefault()
+    const existingOutputs = [...this.state.output]
+
+    existingOutputs.push(<h4 key={shortid.generate()}>{this.props.resourceTemplate.resourceLabel}</h4>)
+    const result = this.populatePropertyTemplates()
+    this.setState({ output: existingOutputs.concat(result)})
   }
 
   handleMintUri = (event) => {
     event.preventDefault()
   }
 
+  populatePropertyTemplates = () => {
+    const newOutput = []
+    this.props.resourceTemplate.propertyTemplates.map((property) => {
+      const keyId = shortid.generate()
+      const newReduxPath = [...this.props.reduxPath]
+      newReduxPath.push(keyId)
+      newOutput.push(<PropertyTemplateOutline
+                      propertyTemplate={property}
+                      rtId={this.props.resourceTemplate.id}
+                      reduxPath={newReduxPath}
+                      key={keyId} />)
+    })
+    return newOutput
+  }
+
   render() {
+    const isAddDisabled = !isRepeatable(this.props.isRepeatable)
```

Comment:
yeah, this would totally read better if the new method was named "templateBoolean()" or "isTrue" or something

Reply:
👍 now changed